README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
 | Declarative test definition                                                               | ✔️       | ✔️                                                           | ✔️                                                                |
 | Monadic test definition                                                                   | ✔️       | ✔️                                                           | ✖️                                                                |
 | Safe test execution                                                                       | ✔️       | ✔️                                                           | ✔️                                                                |
-| Parallel test execution                                                                   | ✔️       | ✔️                                                           | ?                                                                |
-| Parallel or sequential test-group execution                                               | ✔️       | ✔️                                                           | ?                                                                |
+| Parallel test execution                                                                   | ✔️       | ✔️                                                           | ✔️                                                                |
+| Parallel or sequential test-group execution                                               | ✔️       | ✔️                                                           | ✔️                                                                |
 | Automatic test discovery [1]                                                              | ✔️       | ✔️                                                           | ✔️                                                                |
-| First-class support for pure tests                                                        | ✔️       | ✔️                                                           | ✖️                                                                |
+| First-class support for pure tests                                                        | ✔️       | ✔️                                                           | C                                                                |
```

Comment:
I've made this `C` because it's likely possible (I'm not sure) but there is no provider that does this.